main.cpp

```diff
@@ -234,26 +298,35 @@ Thing 3) farm
 
 struct Farm
 {
-    //number of chickens
+    
     int numberOfChickens = 20;
-    //number of cows
     int numberOfCows = 5;
-    //number of workers
     int numberOfWorkers = 1;
-    //the amount of feed per week
     float amountOfFeedPerWeek = 33.3f;
-    //the amount of milk produced
     double amountOfMilkProduced = 22.2;
 
-    //grow veg
-    void growVeg(std:: string vegType, int carrots);
-    //rear animals
+    
+    void growVeg(std:: string vegType, bool carrots);
     void rearAnimals(bool hasChickens);
-    //produce milk
-    void produceMilk(int numberOfLitres = 100);
-
+    void produceMilk(bool amountOfMilk);
 };
 
+void Farm::growVeg(std:: string vegType, bool carrots)
+{
+    (vegType = carrots);
+}
+
+void Farm::rearAnimals(bool hasChickens)
+{
+    hasChickens = true;
+}
+
+void Farm::produceMilk(bool amountOfMilk)
+{
+   (amountOfMilk = true);
+}
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise